packages/@livestore/common/src/materializer-helper.ts

```diff
@@ -44,6 +44,13 @@ export const getExecStatementsFromMaterializer = ({
     ? undefined
     : Schema.encodeUnknownSync(eventDef.schema)(event.decoded!.args)
 
+  // Extract clientId from the event object to pass to materializer
```

Comment:
With the new simplified type signature for `event`, I don't think this checking is needed any longer and `clientId` should always be there, no?